inference-engine/src/mkldnn_plugin/nodes/mkldnn_deconv_node.cpp

```diff
@@ -240,11 +240,11 @@ void MKLDNNDeconvolutionNode::initPadding(std::shared_ptr<ngraph::Node> op, cons
     }
 }
 
-VectorDims MKLDNNDeconvolutionNode::computeInShapeByOutShape(const std::vector<int32_t>& outSpDims,
-                                                             const ov::CoordinateDiff& pb,
-                                                             const ov::CoordinateDiff& pe) const {
+VectorDims MKLDNNDeconvolutionNode::computeOptInDummyShape(const std::vector<int32_t>& outSpDims,
```

Comment:
Nit. It is better to name it `computeOptimalInDummyShape` since Opt usually means optional.